.gitignore

```diff
@@ -6,3 +6,6 @@ internal/util/version/version.txt
 
 # for now
 **/testdata/fuzz
+
+# idea
+.idea
```

Comment:
This file should contain repository-specific ignore patterns. Please use global ignore file for editors, etc: https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer